metro.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config');
+
+const { makeMetroConfig } = require('@rnx-kit/metro-config');
+module.exports = mergeConfig(
+  getDefaultConfig(__dirname),
+  makeMetroConfig({
```

Comment:
`@rnx-kit/metro-config` automatically loads the default config. You don't have to import `@react-native/metro-config` here.

Reply:
Ah, I can update my web view PR then too.